documentation/wasted_openAPI_specification.yaml

```diff
@@ -0,0 +1,377 @@
+openapi: 3.0.3
+info:
+  title: Wasted API - OpenAPI 3.0
+  description: API specification document of the Wasted API
+  version: 1.0.0
+servers:
+  - url: http://localhost:8080/api
+security:
+  - bearerAuth: [] 
+
+paths:
+  /user:
+    get:
+      tags:
+        - Users
+      summary: Returns a list of users.
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/User'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    post:
+      tags:
+        - Users
+      summary: Creates a new user.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/User'
+      responses:
+        '201':
```

Comment:
The status code we have now setup for creating a user is 200

Reply:
Changed both to 200